### ISSUE TITLE
HTML5 video controls inaccessible 

### ISSUE THREAD

**Issue by dcp3450** [cubiq/iscroll#928]
I have a carousel that uses HTML5 videos. The controls can be seen but not clicked/tapped. I've added "tap:true" and "click:true" to my scroller but it has no affect. Placing a video outside the iscroll container fixes the issue.

**Comment by Narthur** [comment]
I believe I may be experiencing a similar issue with a search box.

**Comment by 4ants** [comment]
Is there a solution for this?
Also, flash player like jwplayer is inaccessible (user cannot click controls inside player).

**Comment by dcp3450** [comment]
I had to create my own html5 player controls. I tried using third party
players just like you did and had the same issue. Once I created my own
controls it worked fine. One issue I did run into was the fullscreen
option. Once the video went fullscreen I lost my custom controls. To fix
this, when I clicked the fullscreen option I set the video to "position:
fixed; top: 0; left: 0; width: 100%; height: 100%;" to simulate a
fullscreen. Looks and works fine for me.

**Comment by Allen-su** [comment]
@4ants  set preventDefault:false

**Comment by 4ants** [comment]
Yes, but documentation says that this action can cause unexpected results.
So I added this:

```
lang=javascript
preventDefaultException: {
    tagName: /^(INPUT|TEXTAREA|BUTTON|SELECT|OBJECT)$/
}
```

**Comment by Allen-su** [comment]
I had exactly same issue on android, but I put the HTML5 video label in an iframe label  to fix it . But I don't know why
